### PR TITLE
Mapping project minor updates

### PR DESCRIPTION
**Pull request by bmegli**
Listed mouse and keys for scene interaction
Linked to x3dom supported browsers
Explained relationship between X3D/W3D/W3C/X3DOM/html in one sentence (!)
Capitalized all the technologies correctly

### CODE REVIEW

File: projects/_posts/2016-08-07-Mapping.md [hunk 1]
```diff
@@ -41,15 +41,23 @@ This is my personal project. While I am quite happy to share the results, please
...
 ## Examples
 
-The examples are X3D models. 
+The examples are [X3D] models. 
 
 
-**_You can rotate, move and zoom with the mouse._**
+**_You can interact with the scenes with your mouse and keyboard:_**
+
+- rotation with LMB
```
Comment (bmegli) [inline projects/_posts/2016-08-07-Mapping.md]: Thank you!

**Comment by bmegli** [comment]
Added html key formatting as suggested by @WasabiFan

**Comment by bmegli** [comment]
And finally example phase I result which is also interesting and only 40 kb ;-)

**Comment by bmegli** [comment]
Even one more link to keyboard/mouse interaction, changing modes of navigation etc

**Comment by bmegli** [comment]
Fixed incorrect information about command key on mac.

**Comment by WasabiFan** [comment]
@bmegli Is this PR mostly finalized? Once it's done, I will make a final review pass and then merge.

**Comment by bmegli** [comment]
Yes @WasabiFan, it's done.

 I am sorry for commiting/pushing like crazy. I have been using SVN for years and this can be seen in my naughty behaviour.

I am reading on git hevily and adapting to it, one complete pull request next time.

**Comment by WasabiFan** [comment]
No problem; I just want to make sure I don't merge partial code. This looks good, so I'll merge it.

**Pull request merged**